examples/deepseek/conf/serve/deepseek_v3.yaml

```diff
@@ -1,14 +1,13 @@
-model_args:
-  vllm_model:
-    model-tag: /models/deepseek_v3/model # path of weight of deepseek v3
-    tensor-parallel-size: 8
-    pipeline-parallel-size: 2
-    gpu-memory-utilization: 0.9
-    max-num-seqs: 256
+- model: vllm_model
```

Comment:
The `model: vllm_model` has the same key with `model` in engine_args.

Reply:
changed to serve_id